internal/view/helpers.go

```diff
@@ -261,3 +261,24 @@ func linesWithRegions(lines []string, matches fuzzy.Matches) []string {
 	}
 	return ll
 }
+
+func mapAssign(dst, src map[string]any) map[string]any {
+	out := make(map[string]any, len(dst))
+	for k, v := range dst {
+		out[k] = v
+	}
+
+	for k, v := range src {
+		if srcMap, ok := v.(map[string]any); ok {
+			if dstVal, exists := out[k]; exists {
+				if dstMap, ok := dstVal.(map[string]any); ok {
+					out[k] = mapAssign(dstMap, srcMap)
```

Comment:
Hi Jayson. We could to this in a different PR... but making a note here.
I think it would be very useful to highlight the deltas ie standard vs overridden values in the output. 
What do you think?